tests/exporters/exporters_utils.py

```diff
@@ -74,6 +74,7 @@
         "hf-internal-testing/tiny-random-vision_perceiver_conv": ["image-classification"],
     },
     # "rembert": "google/rembert",
+    "poolformer": "sail/poolformer_s12",
```

Comment:
```suggestion
    "poolformer": "hf-internal-testing/tiny-random-PoolFormerModel",
```

Can you try with this one (sometimes the models from hf-internal-testing that were autogenerated are not good) to keep the tested models small?

Reply:
@fxmarty sure I will test.

Reply:
if you want you can just "commit suggestion" and I will run the workflows.

Reply:
@fxmarty Just updated
![22ss](https://user-images.githubusercontent.com/51019420/209800990-d5ea7c14-5241-4125-af9d-d94823b11bcc.jpg)